Sources/XcbeautifyLib/Regex.swift

```diff
@@ -15,4 +15,30 @@ class Regex {
         let fullRange = NSRange(string.startIndex..., in: string)
         return matcher?.rangeOfFirstMatch(in: string, range: fullRange).location != NSNotFound
     }
+    
+    func capturedGroups(string: String) -> [String] {
+        var results = [String]()
+
+        var regex: NSRegularExpression
```

Comment:
would this also work?

```suggestion
        let regex: NSRegularExpression
```